library/plugins/influxdb/influxdb2-plugin/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id("talaiotPlugin")
+}
+
+talaiotPlugin {
+    idPlugin = "io.github.cdsap.talaiot.plugin.influxdb2"
+    artifact = "influxdb"
```

Comment:
The id for the plugin is correct `"io.github.cdsap.talaiot.plugin.influxdb2` and covers the Gradle Plugin Portal disrtribution, but I'm not sure about the artifact name. I need to confirm the maven coordinates published for the plugins.

Reply:
Changed to "influxdb2"